promise/tests/test-promise.js

```diff
@@ -131,7 +131,7 @@ o.spec("promise", function() {
 				callAsync(function() {resolve(promise)})
 			})
 
-			promise.then(null, done)
+			promise.then(null, () => done())
```

Comment:
For this, only in places where you have a diff. The rest should be flagged in a separate issue, if you'd like to file one for that. (The polyfill tests *should* be runnable in an ES5 environment for obvious reasons.)